dss/storage/bundles.py

```diff
@@ -119,3 +120,41 @@ def _latest_version_from_object_names(object_names: typing.Iterator[str]) -> str
             version = current_version
 
     return version
+
+
+def list_available_uuids(replica: str = None, prefix: str = None, per_page: int = PerPageBounds.per_page_max,
```

Comment:
types for `prefix`, `search_after`, and `token` should be `typing.Optional[str]`

Reply:
complete